src/section-slope-intercept-form.mbx

```diff
@@ -19,7 +19,7 @@
 
     <objectives>
         <title>PCC Course Content and Outcome Guide</title>
-        <ul>
+        <ul cols="3">
```

Comment:
I don't think we should put cols on the objectives lists. Right now, in HTML, each item is just something like "Item C.1.1:8.d" and the PDF omits these entirely. But there will come a time when the actual text of the CCOG item is printed (and not hidden in a knowl). It may only happen for teacher's editions. But in that case, cols=3 will be bad.

Reply:
Hi Alex:

I was simply following examples from other sections. This is the only section in MTH60 without cols="3". I thought you forgot this section.

You can choose to get rid of all cols from all sections.

Carl Yao

Reply:
Oh, I see. Back when these were still present in the PDF, I probably used cols to save paper. But then just omitting them from the PDF saved even more paper :)  I'll merge this and later uniformly remove them all.